resource/resource.go

```diff
@@ -19,10 +19,17 @@
 // and stylesheets.
 package resource
 
-// When changing files included here, you may add the `-debug` flag, which will
-// avoid embedding any files (but instead passes them through from disk). This
-// avoids having to recompile before testing changes.
-
+// If you add any files to the ui folder, you'll need to generate and make build
```

Comment:
in the non-debug case, it affects *changing* files as well, not just adding as per the comment.